widgets/HTTP-batchsource.json

```diff
@@ -68,6 +68,19 @@
             "default": "json"
           }
         },
+        {
+          "widget-type": "number",
+          "label": "Sample Size",
+          "name": "sampleSize",
+          "widget-attributes": {
+            "default": "1000",
```

Comment:
maybe better to default it to a lower number like 100. 1000 is often much more than the number of results in a single page.

Reply:
Changed all occurrence to default to 100;